### DISCUSSION TITLE
How do I upsert multiple records

### DISCUSSION THREAD

**Discussion by milestones95** [supabase/supabase#2174] [Questions] [answered]
Hi,

I have a table called Orders that has the "order_id" column and "status" column. I'd like to be able to select multiple orders at once on the front end and give them the same status. How can I do that? Thanks!

**Comment by inian** [comment, marked as answer]
If you just want to update the records you can do this @milestones95 
```js
supabase.from('orders')
.update({ status: "newstatus" })
.in('id', [1, 3, 4])
```

If you want to upsert, you can modify the above example to use [upsert](https://supabase.io/docs/reference/javascript/upsert)

**Reply by milestones95** [reply]
Thanks!! You're awesome

**Reply by milestones95** [reply]
How would I get the distinct count of all the orders based on a certain status?
